### DISCUSSION TITLE
Calendar component for mobile

### DISCUSSION THREAD

**Discussion by LeoPote** [ant-design/ant-design-mobile#6534] [Q&A]
Hi there,

Is there any possibility to set a locale for the Calendar component for mobile?

Thank you

**Comment by moonlitusun** [comment]
[Internationalization](https://mobile.ant.design/guide/i18n)

```tsx
import enUS from 'antd-mobile/es/locales/en-US';

 <ConfigProvider locale={enUS}>
   <Calendar />
</ConfigProvider>
```
![CleanShot 2024-03-13 at 15 54 10](https://github.com/ant-design/ant-design-mobile/assets/22940470/84ea5dc1-46af-4e33-811c-3a6f7db749dc)
